website/docs/r/variable_set.html.markdown

```diff
@@ -0,0 +1,118 @@
+---
+layout: "tfe"
+page_title: "Terraform Enterprise: tfe_variable_set"
+sidebar_current: "docs-resource-tfe-variable-set"
+description: |-
+  Manages variable sets.
+---
+
+# tfe_variable_set
+
+Creates, updates and destroys variable sets.
+
+## Example Usage
+
+Basic usage:
+
+```hcl
+resource "tfe_organization" "test" {
+  name  = "my-org-name"
+  email = "admin@company.com"
+}
+
+resource "tfe_workspace" "test" {
+  name         = "my-workspace-name"
+  organization = tfe_organization.test.id
+}
+
+resource "tfe_variable_set" "test" {
+  name         = "Test Varset"
+  description  = "Some description."
+  global       = false
+  organization = tfe_organization.test.id
+  workspaces   = [tfe_workspace.test.id]
+}
+
+resource "tfe_variable" "test" {
+  key          = "seperate_variable"
+  value        = "my_value_name"
+  category     = "terraform"
+  description  = "a useful description"
+  variable_set = tfe_variable_set.test.id
+}
+
+resource "tfe_variable" "test" {
+  key          = "another_variable"
+  value        = "my_value_name"
+  category     = "env"
+  description  = "an environment variable"
+  variable_set = tfe_variable_set.test.id
+}
+```
+
+Creating a global variable set:
+
+```hcl
+resource "tfe_organization" "test" {
+  name  = "my-org-name"
+  email = "admin@company.com"
+}
+
+resource "tfe_variable_set" "test" {
+  name         = "Global Varset"
+  description  = "Variable set applied to all workspaces."
+  global       = true
+  organization = tfe_organization.test.id
+}
+
+resource "tfe_variable" "test" {
+  key             = "seperate_variable"
+  value           = "my_value_name"
+  category        = "terraform"
+  description     = "a useful description"
+  variable_set_id = tfe_variable_set.test.id
+}
+
+resource "tfe_variable" "test" {
+  key             = "another_variable"
+  value           = "my_value_name"
+  category        = "env"
+  description     = "an environment variable"
+  variable_set_id = tfe_variable_set.test.id
+}
+```
+
+Creating a variable set that is applied based on workspace tags:
+
+```hcl
+resource "tfe_organization" "test" {
+  name  = "my-org-name"
+  email = "admin@company.com"
+}
+
+data "tfe_workspace_ids" "prod-apps" {
+  tag_names    = ["prod", "app", "aws"]
+  organization = tfe_organization.test.id
+}
+
+resource "tfe_variable_set" "test" {
+  name         = "Tag Based Varset"
+  description  = "Variable set applied to workspaces based on tag."
+  organization = tfe_organization.test.id
+  workspaces   = tfe_workspace_ids.prod-apps.ids
```

Comment:
Ditto on computed attributes showing up in the configuration (we should only have arguments)

Reply:
I may be wrong here and computed attributes that are also arguments are totally valid.